test/Tests/src/Data/Text/Regex_Spec.enso

```diff
@@ -513,6 +513,15 @@ spec =
             match.utf_16_span 5 . should_fail_with No_Such_Group
             match.utf_16_span "nonexistent" . should_fail_with No_Such_Group
 
+    Test.group "Match.to_display_text" <|
+        pattern = Regex.compile "(.. .. )(?<letters>.+)()??(?<empty>)??"
+        input = "aa ab abc a bc bcd"
+        match = pattern.match input
+        match . should_be_a Match
+
+        Test.specify "should not error" <|
```

Comment:
```suggestion
        Test.specify "should not error" <|
            match = pattern.match input
            match . should_be_a Match

```

We should not use `should_be_a` outside of `Test.specify` - it may cause weird errors on failure, it is only meant to be used inside of a test case

Reply:
@radeusgd It's worth noting that all the other tests use `should_be_a` outside the `Test.specify`, so maybe a task should be created for finding and fixing those across every test file

Reply:
Oh, sorry, I did not realize this was the case in other tests. Then indeed it seems totally fine to leave it as-is and we can revise at some point.